application/F3DStarter.cxx

```diff
@@ -440,6 +440,20 @@ class F3DStarter::F3DInternals
       std::to_string(maxNumberingAttempts) + " attempts");
   }
 
+  static void PrintLoggingMap(
+    const std::map<std::string, std::tuple<std::string, fs::path, std::string, std::string>>&
```

Comment:
Maybe create an alias for `std::tuple<std::string, fs::path, std::string, std::string>`?